PowerShell/JumpCloud Module/Public/Commands/Get-JCCommand.ps1

```diff
@@ -1,10 +1,34 @@
 
 Function Get-JCCommand ()
 {
-    [CmdletBinding(DefaultParameterSetName = 'ReturnAll')]
+    [CmdletBinding(DefaultParameterSetName = 'SearchFilter')]
 
     param
     (
+
+    #### NEW PARAMS
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'The command to execute on the server.')]
```

Comment:
I'm thinking we should change this to something a bit more descriptive for each Help Message. On Get-JCUser we user: "The Username of the JumpCloud user you wish to search for." to kind of inform the user that this is a search field. I'd like to do the same here. 

How do you feel about the following:
The command body text of the JumpCloud Command you wish to search for. Ex. `Get-JCCommand -Command "*Get-LocalUser*`" would return commands that have the Get-LocalUser text in the command body.